pyaedt/hfss3dlayout.py

```diff
@@ -2202,3 +2202,50 @@ def get_dcir_element_data_via(self, setup_name):
             else:
                 df.merge(df_tmp, left_index=True, right_index=True, how="outer")
         return df
+
+    @pyaedt_function_handler
+    def show_extent(self, show=True):
+        """Show or hide extent in a HFSS3dLayout design.
+
+        Parameters
+        ----------
+        show: bool, optional
```

Comment:
This should fix the documentation failure.
```suggestion
        show : bool, optional
```